src/handlers/__tests__/displayNameHandler-test.js

```diff
@@ -70,17 +70,30 @@ describe('defaultPropsHandler', () => {
     expect(documentation.displayName).not.toBeDefined();
   });
 
-  it('ignores non-literal names', () => {
-    var definition = expression('({displayName: foo.bar})');
-    expect(() => displayNameHandler(documentation, definition)).not.toThrow();
-    expect(documentation.displayName).not.toBeDefined();
-
+  it('infers the displayName with es6 class', () => {
+    var definition = expression('({displayName: "FooBar"})');
+    displayNameHandler(documentation, definition);
+    expect(documentation.displayName).toBe('FooBar');
     definition = statement(`
       class Foo {
-        static displayName = foo.bar;
+
       }
     `);
-    expect(() => displayNameHandler(documentation, definition)).not.toThrow();
-    expect(documentation.displayName).not.toBeDefined();
+    displayNameHandler(documentation, definition);
+    expect(documentation.displayName).toBe('Foo');
+  });
+
+  it('infers the displayName with stateless functional component', () => {
+    var definition = expression('({displayName: "FooBar"})');
+    displayNameHandler(documentation, definition);
+    expect(documentation.displayName).toBe('FooBar');
+
+    definition = statement(`
+      var Foo = () => {
```

Comment:
Okay so, I still want to be able to infer the name and assign the as the displayName of stateless functional component.

What would you think about exporting ```resolveName()``` from ```/getMemberExpressionValuePath.js```, importing it into ```displayNameHandler``` and using that to grab the variable definition display names?